apps/src/code-studio/pd/application_dashboard/application_dashboard.jsx

```diff
@@ -34,30 +43,30 @@ const ApplicationDashboard = ()=> (
         breadcrumbs="Summary"
         component={Summary}
       />
-      <Route
-        path="csd_facilitators"
-        breadcrumbs="CSD Facilitator Applications"
-        component={QuickView}
-        title="CSD Facilitator Applications"
-      />
-      <Route
-        path="csp_facilitators"
-        breadcrumbs="CSP Facilitator Applications"
-        component={QuickView}
-        title="CSP Facilitator Applications"
-      />
-      <Route
-        path="csd_teachers"
-        breadcrumbs="CSD Teacher Applications"
-        component={QuickView}
-        title="CSD Teacher Applications"
-      />
-      <Route
-        path="csp_teachers"
-        breadcrumbs="CSP Teacher Applications"
-        component={QuickView}
-        title="CSP Teacher Applications"
-      />
+      {
+        _.flatten(Object.keys(paths).map((path, i) => {
+          return [
+            (
+              <Route
+                key={`detail_${i}`}
+                path={`${path}/(:applicationId)`}
+                breadcrumb={`${paths[path]},Application Details`}
+                component={DetailView}
+                title={'Application Detail'}
```

Comment:
This doesn't have to change for this PR, but going forward apparently a better way to pass params to components like DetailView inside a route is to use higher-order components rather than tacking on props to the Route. (my bad for not setting up this better pattern originally)

`const quickViewForAppType = (title, ...props) => {
  <QuickView
    title={title}
    {...props}
  />
}`

and then it can be called like this:
   `<Route
        path="csd_facilitators"
        breadcrumbs="CSD Facilitator Applications"
        component={quickViewForAppType("CSD Facilitator Applications")}
      />`